tests/RequestTest.php

```diff
@@ -201,4 +201,12 @@ public function testCanHaveHeaderWithEmptyValue()
         $r = $r->withHeader('Foo', '');
         $this->assertEquals([''], $r->getHeader('Foo'));
     }
+
+    public function testCannotHaveEmptyHeaderValues()
```

Comment:
Is there a better name for this? The real error being tested here is that `null` isn’t a valid string. Empty header values (that is: empty strings) are [valid header values](https://github.com/php-http/psr7-integration-tests/pull/30).

Reply:
I rename it to `testCannotHaveHeaderValueWithNullValue`

Reply:
:+1: Thanks! Mostly because I know I was going to get confused about headers again in a month time otherwise 😉